src/module-elasticsuite-catalog/Controller/Adminhtml/Term/Merchandiser/Save.php

```diff
@@ -54,8 +54,14 @@ public function __construct(
      */
     public function execute()
     {
-        $queryId = $this->getRequest()->getParam('query_id');
-        $sortedProducts = $this->getRequest()->getParam('sorted_products', []);
+        $queryId             = $this->getRequest()->getParam('query_id');
+        $sortedProducts      = $this->getRequest()->getParam('sorted_products', []);
+        $blacklistedProducts = $this->getRequest()->getParam('blacklisted_products', []);
+
```

Comment:
Trailing debug instruction to be removed.

Reply:
ooops my bad...